path_config

```diff
@@ -16,6 +16,11 @@ path /status StatusHandler {
     root .;
 }
 
+path /reverse_proxy ReverseProxyHandler {
+    remote_host localhost;
+    remote_port 1234;
```

Comment:
So this is all based on the discussion on this past friday (in case you didn't attend it), and we might need to revisit it. Yes, we do need to be able to configure the remote host (ucla website, for example), but not the port. It doesn't make sense to target a specific port for the remote host, and since our reverse proxy handler is part of our web server, it listens at the same port that is configured using the "port" keyword in the top level of our config file, and not under reverse proxy handler. So our TA's conclusion was that we don't need to configure the port, and she said updated spec will be uploaded on friday night, but nothing so far.